theme/placeholder.scss

```diff
@@ -2,8 +2,15 @@
 // For licensing, see LICENSE.md or http://ckeditor.com/license
 
 .ck-placeholder::before {
+	position: absolute;
+	top: 0;
+	left: 0;
 	content: attr( data-placeholder );
 	cursor: text;
 	color: #c2c2c2;
 	pointer-events: none; // See ckeditor/ckeditor5#469.
 }
+
+.ck-placeholder {
```

Comment:
I'm afraid that we must not do that. This class is assigned to an element which, in the case of inline editors, may be a part of a website with which the editor is integrated. It will also be applied to e.g. nested editables which may also require a certain styling. So this is really unsafe.

Another unsafe thing here is the `pos:abs, top:0; left:0` styling. See e.g. https://jsfiddle.net/n2gzLqhz/2/:

![image](https://user-images.githubusercontent.com/156149/28501989-cc9025a0-6fe8-11e7-8a3f-88a5dac43e33.png)